doc/api/next_api_changes/deprecations.rst

```diff
@@ -422,3 +422,8 @@ Setting the *orientation* of an ``eventplot()`` or `.EventCollection` to "none"
 or None is deprecated; set it to "horizontal" instead.  Moreover, the two
 orientations ("horizontal" and "vertical") will become case-sensitive in the
 future.
+
+Case-insensitive properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for upper or mixed-case property names in `.Artist.set` and
```

Comment:
If `patchA` is now accepted, then this statement is incorrect now, isn't it?

Reply:
It's not accepted now (and wasn't before) (as it was and is still wrongly normalized to patcha), but will be after the deprecation.

Reply:
Right, but this writeup says the opposite (even it didn't work before.)

Reply:
I see, the wording was bad, pushed a different one, how does that look?